v2/apiserver/internal/core/rest/role_assignments_endpoints.go

```diff
@@ -0,0 +1,81 @@
+package rest
```

Comment:
Ditto note here re: potentially updating file names for this and similar sibling files to `project_role_*.go`

Reply:
Renamed